CMakeLists.txt

```diff
@@ -30,6 +30,8 @@ set($ENV{HTM_CPP} ${REPOSITORY_DIR})
 option(FORCE_CPP11 "Force compiler to use C++11 standard." OFF)
 option(FORCE_BOOST "Force compiler to install and use Boost." OFF)
 set(BINDING_BUILD "none" CACHE STRING "Specify the Binding to build 'Python2','Python3' or 'none', default 'none'." )
+set(NTA_LIBC_MUSL OFF CACHE BOOL "Are we using MUSL stdlibc++, or GLIBC?")
```

Comment:
@dkeeney this is a little complicated, is it a correct way to pass command-line `-DNTA_LIBC_MUSL=1` to `src/CMakeLists.txt`?

Reply:
Well, yes that might work.   But it is normally convention to use the option( ) statement for boolean CMake parameters that we want the user to be able to set.
```
option(NTA_LIBC_MUSL "Are we using MUSL stdlibc++? otherwise GLIBC " OFF)
```
The user would then use `-DNTA_LIBC_MUSL` in the CMake command line if it is to be ON.

Reply:
thanks. I must have overlooked the option. Now it seems to work without the extra `if()`s :+1: